apps/flame-defi/app/earn/modules/vault-details/sections/header-section.tsx

```diff
@@ -45,9 +44,22 @@ export const HeaderSection = () => {
                   />
                   <span>{data?.vaultByAddress.asset.symbol}</span>
                 </Badge>
-                <VaultCurators
-                  curators={data?.vaultByAddress.metadata?.curators}
-                />
+                {data?.vaultByAddress.metadata?.curators?.map((it, index) => (
```

Comment:
I can't comment on it, but above, the expression `data?.vaultByAddress?.asset.logoURI` does not need the second optional chain operator